### PR TITLE
Change to rebar project

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: vsn-check.sh [hunk 1]
```diff
@@ -1,23 +0,0 @@
-#!/bin/bash -e
-
-PROJECT_VERSION="$1"
-SNAPPYER_VSN="$2"
-
-APP_VSN=$(erl -noshell -eval '{ok, [{_,_,L}]} = file:consult("src/kafka_protocol.app.src"), {vsn, Vsn} = lists:keyfind(vsn, 1, L), io:format("~s", [Vsn]), halt(0).')
```
Comment (id) [inline vsn-check.sh:6]: why remove this?

File: Makefile [hunk 2]
```diff
@@ -1,23 +1,12 @@
-PROJECT = kafka_protocol
-PROJECT_DESCRIPTION = Kafka protocol erlang library
-PROJECT_VERSION = 1.0.0
 
-EUNIT_OPTS = verbose
-ERLC_OPTS = -Werror +warn_unused_vars +warn_shadow_vars +warn_unused_import +warn_obsolete_guard +debug_info
-CT_OPTS = -ct_use_short_names true
```
Comment (id) [inline Makefile:7]: don't need this in rebar.config?

File: rebar.config.script [hunk 3]
```diff
@@ -1,14 +1,8 @@
 IsRebar3 = erlang:function_exported(rebar3, main, 1),
-IsNoSnappy = case os:getenv("KAFKA_PROTOCOL_NO_SNAPPY") of
-               "1" -> true;
-               _   -> false
-             end,
-case {IsNoSnappy, IsRebar3} of
-  {true, _} ->
-    lists:keyreplace(deps, 1, CONFIG, {deps, []});
-  {false, true} ->
+case IsRebar3 of
```
Comment (id) [inline rebar.config.script:2]: include snappyer for rebar3 only?

File: Makefile (hunk 2 above)
Comment (zmstone) [reply Makefile:7]: we don't have ct in the proj

Comment (id) [reply Makefile:7]: right

File: rebar.config.script (hunk 3 above)
Comment (zmstone) [reply rebar.config.script:2]: this is to downgrade dependency tuple from rebar3 format to rebar format.
not to delete the dependency

File: vsn-check.sh (hunk 1 above)
Comment (zmstone) [reply vsn-check.sh:6]: we used to have version defined in Makefile and .app.src
Now it's in .app.src only.

Comment (id) [reply vsn-check.sh:6]: right, we don't have release for this one

**Review by id — APPROVED** [review]

**Pull request merged**